test/integration/mempool-integration.js

```diff
@@ -3,28 +3,28 @@
 
 'use strict';
 
-const assert = require('./util/assert');
+const assert = require('../util/assert');
 const random = require('bcrypto/lib/random');
-const MempoolEntry = require('../lib/mempool/mempoolentry');
-const Mempool = require('../lib/mempool/mempool');
-const WorkerPool = require('../lib/workers/workerpool');
-const Chain = require('../lib/blockchain/chain');
-const MTX = require('../lib/primitives/mtx');
-const Coin = require('../lib/primitives/coin');
-const KeyRing = require('../lib/primitives/keyring');
-const Address = require('../lib/primitives/address');
-const Outpoint = require('../lib/primitives/outpoint');
-const Script = require('../lib/script/script');
-const Witness = require('../lib/script/witness');
-const CoinView = require('../lib/coins/coinview');
-const MemWallet = require('./util/memwallet');
+const MempoolEntry = require('../../lib/mempool/mempoolentry');
+const Mempool = require('../../lib/mempool/mempool');
+const WorkerPool = require('../../lib/workers/workerpool');
+const Chain = require('../../lib/blockchain/chain');
+const MTX = require('../../lib/primitives/mtx');
+const Coin = require('../../lib/primitives/coin');
+const KeyRing = require('../../lib/primitives/keyring');
+const Address = require('../../lib/primitives/address');
+const Outpoint = require('../../lib/primitives/outpoint');
+const Script = require('../../lib/script/script');
+const Witness = require('../../lib/script/witness');
+const CoinView = require('../../lib/coins/coinview');
+const MemWallet = require('../util/memwallet');
 const ALL = Script.hashType.ALL;
 
 const ONE_HASH = Buffer.alloc(32, 0x00);
 ONE_HASH[0] = 0x01;
 
 const workers = new WorkerPool({
-  enabled: true
+  enabled: process.browser ? false : true
```

Comment:
I haven't dug in deep enough yet to determine why the workers throw errors when running them in the browser. Turning them off allows the tests to pass. Ideally the workers should work fine in the browser, since they do in `bcoin`. @pinheadmz may have some insight into this

Reply:
I'm actually not entirely sure workers ever worked in the browser.
If you go to https://browser.c4yt.io and watch the `network` tab in browser dev tools, you'll see `/worker.js` is never loaded from the web server. [Even though it looks like it should be.](https://github.com/bcoin-org/bcoin/blob/master/browser/src/app.js#L66-L67)

Reply:
Resolved based on https://github.com/bcoin-org/bcoin/commit/0b70a940a3f6e6dc8be13b844cda8e1b9d338a60